proto/viam/service/sensors/v1/sensors.proto

```diff
@@ -10,49 +10,54 @@ option go_package = "go.viam.com/api/service/sensors/v1";
 option java_package = "com.viam.service.sensors.v1";
 
 // A SensorsService services keeps track of all sensors associated with a robot
+// This service is deprecated
 service SensorsService {
   // GetSensors returns the list of all sensors.
   rpc GetSensors(GetSensorsRequest) returns (GetSensorsResponse) {
     option (google.api.http) = {
       get: "/viam/api/v1/service/{name}/sensors"
     };
+    option deprecated = true;
   }
 
   // GetReadings returns the list of readings for all sensors specified.
   rpc GetReadings(GetReadingsRequest) returns (GetReadingsResponse) {
     option (google.api.http) = {
       get: "/viam/api/v1/service/sensors/{name}/readings"
     };
+    option deprecated = true;
   }
 
   // DoCommand sends/receives arbitrary commands
   rpc DoCommand(common.v1.DoCommandRequest) returns (common.v1.DoCommandResponse) {
     option (google.api.http) = {
       post: "/viam/api/v1/service/sensors/{name}/do_command"
     };
+    option deprecated = true;
   }
 }
 
+// The sensor service messages are deprecated
 message GetSensorsRequest {
-  string name = 1;
+  string name = 1 [deprecated = true];
   // Additional arguments to the method
-  google.protobuf.Struct extra = 99;
+  google.protobuf.Struct extra = 99 [deprecated = true];
 }
 
 message GetSensorsResponse {
-  repeated common.v1.ResourceName sensor_names = 1;
```

Comment:
I think that in addition to giving RPCs the deprecated option, [you can do that to entire messages](https://github.com/protocolbuffers/protobuf/issues/1734#issuecomment-234681729), too. Worth doing here? I dunno; what you've got is already pretty thorough.

Reply:
nice! I am not a proto expert, had to google what to actually do here. I don't see the harm in just adding deprecated signifiers everywhere we can.

Reply:
I think it's:
```
message GetSensorsResponse {
    option deprecated = true;
    repeated common.v1.ResourceName sensor_names = 1 [deprecated = true]; 
    ...
}
```
but I've never done this myself and could be wrong.

Reply:
Okay I'm satisfied. I will check with a local version of this API with rdk tomorrow to make sure things work as expected and won't break anything juts by adding a deprecated signifier to proto.